pennylane/wires.py

```diff
@@ -31,6 +31,10 @@ def _process(wires):
         # if input is already a Wires object, just return its wire tuple
         return wires.wire_tuple
 
+    elif isinstance(wires, (Number, str)):
+        # interpret as a single wire
+        return (wires,)
```

Comment:
:+1: The drawbacks of Python treating strings as sequences of characters. Every now and again my code will fail and the output will be
```
['s', 'o', 'm', 'e', ' ', 's', 't', 'r', 'i', 'n', 'g']
```
and I'll know I've done it again.